tests/utils.go

```diff
@@ -2910,14 +2912,20 @@ func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter
 		&expect.BExp{R: prompt},
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: "0"},
-		&expect.BSnd{S: fmt.Sprintf(`echo "nameserver %s" >> /etc/resolv.conf\n`, dnsServerIP)},
+		&expect.BSnd{S: "sudo su\n"},
```

Comment:
the code wont run on fedoras that already have the nameserver in their user-data

Reply:
And on fedora VMs that don't ? I think having a function that has redundant stuff based on the input is even worse than having a function that does redundant stuff always.

Reply:
the sandboxing here is needed thats my point, 
else we need a sleep (not deterministic) or something, because if you take the original code and add
only this:

```
cat /etc/resolv.conf | tail -1
expect the right thing
```
it will pass
but if you
```
expect wrong thing
```
it will fail

and it actually didnt change the file (and even shows permission denided).
so it uses its own strings, and the flow is broken because of the backquote

this is why i think the sandbox is important

Reply:
fedora doesnt rerun it, because the beginning of the function checks if there is already ipv6, and then it doesnt do the 2nd block (ip + route + dns),
anyway that block always fails because the expect isnt formatted well, i added another commit (wip) that fix the expect and remove the dns, as we said that we dont need it actually, and it was added just nice to have, without actual proven need

Reply:
sorry, I didn't mean to push this comment, since I've later realized you were pushing stuff on top to later squash. Couldn't find it afterwards since the conversation was marked as resolved (as as such, the comment tree was collapsed.) 

Since I couldn't find it, I thought I had deleted it.

Forget about this particular question.